src/main/scala/org/broadinstitute/dsde/firecloud/utils/TSVParser.scala

```diff
@@ -14,23 +11,31 @@ case class TSVLoadFile(
 )
 
 object TSVParser {
-  private def parseLine( tsvLine: String, tsvLineNo: Int, nCols: Int ) = {
-    val fields = tsvLine.split("\t", -1) //note we don't trim the line here in case the last element is deliberately empty
-    if( fields.length != nCols ) {
-      throw new RuntimeException("TSV parsing error in line " + tsvLineNo.toString + ": wrong number of fields")
-    }
-    fields.toList
+
+  private lazy val PARSER: CsvParser = {
```

Comment:
nitpick: the only place we use uppercase variables is for constants (and sometimes not even then).